pkg/resourceinterpreter/configurableinterpreter/luavm/lua_test.go

```diff
@@ -188,33 +187,21 @@ func TestAggregateDeploymentStatus(t *testing.T) {
 	oldObj, _ := helper.ToUnstructured(oldDeploy)
 	newObj, _ := helper.ToUnstructured(newDeploy)
 
-	var aggregateItem []map[string]interface{}
-	for _, item := range aggregatedStatusItems {
-		if item.Status == nil {
-			continue
-		}
-		temp := make(map[string]interface{})
-		if err := json.Unmarshal(item.Status.Raw, &temp); err != nil {
-			t.Error(err.Error())
-		}
-		aggregateItem = append(aggregateItem, temp)
-	}
-
 	tests := []struct {
 		name                  string
 		curObj                *unstructured.Unstructured
-		aggregatedStatusItems []map[string]interface{}
+		aggregatedStatusItems []workv1alpha2.AggregatedStatusItem
 		expectedObj           *unstructured.Unstructured
 		luaScript             string
 	}{
 		{
 			name:                  "Test AggregateDeploymentStatus",
 			curObj:                oldObj,
-			aggregatedStatusItems: aggregateItem,
+			aggregatedStatusItems: aggregatedStatusItems,
 			expectedObj:           newObj,
 			luaScript: `function AggregateStatus(desiredObj, statusItems)  
 										for i = 1, #statusItems do    
-											desiredObj.status.readyReplicas = desiredObj.status.readyReplicas + statusItems[i].readyReplicas 
+											desiredObj.status.readyReplicas = desiredObj.status.readyReplicas + 1
```

Comment:
why change it?

Reply:
I didn't get the reason why change it, I think the test should be close to the real scenario.